samples/tls/logging/README.md

```diff
@@ -2,7 +2,24 @@
 
 ## Overview
 
-Communication between the logging components within the cluster always takes place through TLS-enabled connections. This sample demonstrates how to deploy logging with TLS enabled for connections between the user and Kibana. The `TLS_ENABLE` environment variable controls whether connections to Kibana use TLS.
+TLS enablement for SAS Viya logging is divided into two parts:
+
+- **TLS for in-cluster communications**, which is between logging components within
+the cluster (between the Elasticsearch nodes and between Elasticsearch and other logging components). TLS must be enabled on these connections.
+
+- **TLS for communications into the cluster**, which is between a user's 
+browser and Kibana. TLS is optional on these connections, although enabling
+TLS is a best practice.
+
+This sample demonstrates only how to enable TLS for communications into the 
+cluster by deploying logging with TLS enabled for connections between the 
+user and Kibana. The `TLS_ENABLE` environment variable controls whether 
+connections to Kibana use TLS.
+
+In-cluster communications must always take place through TLS-enabled connections. TLS for in-cluster communications is configured using a separate process, 
+either automatically (using cert-manager) or manually. See [Notes_on_using_TLS](../../../logging/Notes_on_using_TLS.md) for information about configuring TLS for 
+in-cluster communications.
+
```

Comment:
See note above about reorganizing the paragraphs...think it makes sense to do that here as well.